app/domain/authentication/authn_jwt/authentication_parameters.rb

```diff
@@ -0,0 +1,24 @@
+module Authentication
+  module AuthnJwt
+    # Data class to store data regarding jwt token that is needed during the jwt authentication process
+    class AuthenticationParameters
+      attr_writer :jwt_identity
```

Comment:
Why this attribute is granted write access?
This class should be immutable as it is data class.
Init it with all its parameters in the ctor and never change it, ever.
Also, this attribute is write-only, no one can read it; looks very weird...

Reply:
Changed to accessor so will be able to read.
This is something that is not defined during ctor because ctor runs after `validate_and_decode` and this runs after `get_jwt_identity`